### DISCUSSION TITLE
Hashify Power Automate - populate the 'value parameter'

### DISCUSSION THREAD

**Discussion by televisi** [dwin/hashify#14] [General]
Hi @dwin,

Great script!

Looking at the PowerAutomate connector page (https://learn.microsoft.com/en-au/connectors/hashifyip/); I suspect you were the one that created the script too? 

As I'm interested in using your script to run a checksum against msi file (around 25MB file size).

I noticed the "Post an array for MD5" require 2 x mandatory parameters:

- Value
- File

As I'd like to get MD5 for a file, I suspect I just need to use the file contents populated on the **File** field only? 
How about the **Value** field, what should I populate it with?

![image](https://github.com/dwin/hashify/assets/4546680/5ec59ca5-c0ab-45d9-930d-2faffd226075)

Thanks

**Comment by dwin** [comment]
I did not create the script and I'm not familiar with Power Automate.

**Reply by televisi** [reply]
@dwin Thanks for your reply!
Sorry, didn't realise that you were not the creator of the script.

I'll reach out to Troy based on the [Ms connector page](https://learn.microsoft.com/en-au/connectors/hashifyip/).

Thanks heaps!
